magiskmodule/post-fs-data.sh

```diff
@@ -0,0 +1,5 @@
+
+# bind mount all odm files
+for file in $(find "$MODPATH"/system/odm -type f); do
+    mount -o bind "$file" $(sed "s|$MODPATH/system||g" <<< "$file")
+done
```

Comment:
```suggestion
```
This won't be needed anymore when [Magisk#6588](https://github.com/topjohnwu/Magisk/pull/6588) is merged.

Reply:
Included in Magisk since canary `25207`.